templates/Web/Projects/VueDefault/.template.config/template.json

```diff
@@ -0,0 +1,94 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": ["Universal"],
+  "name": "VueDefault",
+  "groupIdentity": "wts.Proj.VueDefault",
+  "identity": "wts.Proj.VueDefault",
+  "description": "",
+  "tags": {
+    "language": "Any",
+    "type": "project",
+    "wts.type": "project",
+    "wts.projecttype": "FullStackWebApp",
+    "wts.frontendframework": "Vue",
+    "wts.backendframework": "all",
+    "wts.platform": "Web",
+    "wts.version": "1.0.0",
+    "wts.displayOrder": "1"
+  },
+  "sourceName": "wts.VueDefault",
+  "preferNameDirectory": true,
+  "PrimaryOutputs": [],
+  "symbols": {
+    "wts.userName": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_UserName"
+    },
+    "wts.wizardVersion": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_Wizard.Version"
+    },
+    "wts.templatesVersion": {
+      "type": "parameter",
+      "dataType": "string",
+      "replaces": "Param_Templates.Version"
+    },
+    "wts.generationProjectType": {
+      "type": "parameter",
+      "replaces": "Param_ProjectType"
+    },
+    "wts.generationFramework": {
+      "type": "parameter",
+      "replaces": "Param_Framework"
+    },
+    "wts.generationPlatform": {
+      "type": "parameter",
+      "replaces": "Param_Platform"
+    },
+    "copyrightYear": {
+      "type": "generated",
+      "generator": "now",
+      "replaces": "1976",
+      "parameters": {
+        "format": "yyyy"
+      }
+    }
+  },
+  "postActions": [
+    {
+      "description": "Add Vue Dependencies",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'bootstrap': '^4.3.1', 'core-js': '^2.6.5', 'vue': '^2.6.6', 'vue-router': '^3.0.4'}",
+        "key": "dependencies",
+        "jsonPath": "package.json"
+      },
+      "continueOnError": false
+    },
+    {
+      "description": "Add Vue Dev Dependencies",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'@vue/cli-plugin-babel': '^3.5.0', '@vue/cli-plugin-eslint': '^3.5.0', '@vue/cli-service': '^3.5.0',  'babel-eslint': '^10.0.1', 'eslint': '^5.8.0', 'eslint-plugin-vue': '^5.0.0', 'vue-template-compiler': '^2.5.21'}",
+        "key": "devDependencies",
+        "jsonPath": "package.json"
+      },
+      "continueOnError": false
+    },
+    {
+      "description": "Add Vue Build Scripts",
+      "manualInstructions": [],
+      "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
+      "args": {
+        "dict": "{'start': 'concurrently \"vue-cli-service serve\" \"nodemon server/server.js\"', 'build': 'vue-cli-service build && mv build server', 'lint': 'vue-cli-service lint'}",
```

Comment:
By adding the `start` script into the front-end frameworks only, we are assuming here that the back-end framework is always Node.js. However, as we plan to integrate flask, the start script would actually change to: `concurrently \"vue-cli-service serve\" \"nodemon server/server.js\"`. 
I was just curious if you have some ideas on how would we construct a script command that needs to know both front-end as well as back-end framework.

Reply:
this is very true. I feel like this should be dealt with in a separate PR though since this affects all 3 frontend frameworks. We could have a composition template that adds into "scripts" section. Detecting the frontend + backend selection. Shouldn't be too hard.

Reply:
Can you make an issue for this @njindal239 ?

Reply:
Sure, I will do it.

Reply:
Cool I'll assign it to myself and I'll have something like this done by EOD Today/Friday if thats okay?